src/app/public/modules/tabs/tab-active-change.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  SkyTabIndex
+} from './tab-index';
+
+/**
+ * @internal
+ */
+export interface SkyTabActiveChange {
+
+  active: boolean;
+
+  tabIndex?: SkyTabIndex;
```

Comment:
I left it optional because the `activeChange` emitter is a `BehaviorSubject` and needed an initializing value (but a tab index has not yet been set). But you bring up a good point; I'll just set the `BehaviorSubject` `tabIndex` to undefined instead.